target/linux/ramips/dts/mt7621_xiaoyu_xy-c5.dts

```diff
@@ -0,0 +1,105 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaoyu,xy-c5", "mediatek,mt7621-soc";
+	model = "XiaoYu XY-C5";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "xy-c5:green:work";
```

Comment:
or a "work" led? led node name should be the same as its label.

Reply:
![IMG_20190915_080428](https://user-images.githubusercontent.com/6256701/64915083-8ef28900-d791-11e9-9d10-874901af100f.jpg)

yes, WORK led, should be indicator USB
01_leds, `ucidef_set_led_usbport` ?

Reply:
You can set USB indicator directly in DTS. Just have a look at other DTS files for examples.
(Supposed that this WORK led should be used a USB led)